apps/explorer/src/components/validator-map/ValidatorMap.tsx

```diff
@@ -30,23 +31,26 @@ const DATE_FILTER_TO_WINDOW = {
 export default function ValidatorMap() {
     const [dateFilter, setDateFilter] = useDateFilterState('D');
 
-    const { data } = useQuery(['validator-map', dateFilter], async () => {
-        const res = await fetch(
-            `${HOST}/location?${new URLSearchParams({
-                version: 'v2',
-                window: DATE_FILTER_TO_WINDOW[dateFilter],
-            })}`,
-            {
-                method: 'GET',
-            }
-        );
-
-        if (!res.ok) {
-            return [];
+    const { data, isLoading, isSuccess, isError, error } = useQuery(
+        ['validator-map', dateFilter],
+        async () => {
+            const res = await fetch(
+                `${HOST}/location?${new URLSearchParams({
+                    version: 'v2',
+                    window: DATE_FILTER_TO_WINDOW[dateFilter],
+                })}`,
+                {
+                    method: 'GET',
+                }
+            );
+
+            return res.json() as Promise<NodeLocation[]>;
         }
+    );
 
-        return res.json() as Promise<NodeLocation[]>;
-    });
+    if (isError) {
+        console.error('Error in Validator Map data retrieval', error);
```

Comment:
Don't log in the render method. IMO you don't need this, but if you want to keep it either move this to an effect, or use the `onError` function that you can pass to `useQuery`.